### PR TITLE
DAC6-2727 : WhereDoYouLive page added

### PR DESCRIPTION
**Pull request by AashuChauhanHmrc**

### CODE REVIEW

File: app/models/IndWhereDoYouLive.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.RadioItem
+
+sealed trait IndWhereDoYouLive
+
+object IndWhereDoYouLive extends Enumerable.Implicits {
+
+  case object Wheredoyoulive extends WithName("where-do-you-live") with IndWhereDoYouLive
+  case object Wheredoyoulive2 extends WithName("where-do-you-live-2") with IndWhereDoYouLive
```
Comment (matthewharris1) [inline app/models/IndWhereDoYouLive.scala:28]: Can this be given a more specific name to differentiate what both objects are doing?

File: conf/messages.en [hunk 2]
```diff
@@ -390,3 +390,10 @@ whatIsYourUTR.error.required = Enter your Unique Taxpayer Reference
 whatIsYourUTR.error.invalidFormat = Unique Taxpayer Reference must include 10 numbers, for example 1234567890
 whatIsYourUTR.error.invalid = Unique Taxpayer Reference must only include numbers 0 to 9
 whatIsYourUTR.change.hidden = WhatIsYourUTR
+
+indWhereDoYouLive.title = Where do you live?
+indWhereDoYouLive.heading = Where do you live?
+indWhereDoYouLive.where-do-you-live = The UK, Jersey, Guernsey or the Isle of Man
+indWhereDoYouLive.where-do-you-live-2 = Somewhere else
+indWhereDoYouLive.checkYourAnswersLabel = Where do you live?
+indWhereDoYouLive.error.required = 	Select where you live
```
Comment (matthewharris1) [inline conf/messages.en:399]: Whitespace before error message needs removed

**Review by matthewharris1 — CHANGES_REQUESTED** [review]

File: app/models/IndWhereDoYouLive.scala (hunk 1 above)
Comment (AashuChauhanHmrc) [reply app/models/IndWhereDoYouLive.scala]: fixed

File: conf/messages.en (hunk 2 above)
Comment (AashuChauhanHmrc) [reply conf/messages.en]: fixed

**Comment by chriswaugh** [comment]
Closing PR. This page was redesigned and implemented in a separate PR - https://github.com/hmrc/crs-fatca-registration-frontend/pull/70

**Pull request closed without merging**